src/main/java/com/example/bookstore/entities/Book.java

```diff
@@ -23,6 +23,8 @@
 
     private String description;
 
+    public Book() {}
+
     public Book(String isbn, String title, String authors, String publisher, String image, String description) {
```

Comment:
du coup dans ce constructeur tu peux faire référence au premier pour "l'étendre" avec this();

Reply:
done